src/Client/ConfigWindow.java

```diff
@@ -2751,6 +2759,20 @@ public void addWorldFields(int i) {
     worldNamesJTextFields.get(i).setPreferredSize(new Dimension(200, 28));
     worldNamesJTextFields.get(i).setAlignmentY((float) 0.75);
     worldListTitleTextFieldContainers.get(i).add(worldNamesJTextFields.get(i), cR);
+    
+    cR.weightx = 0.2;
+    cR.gridwidth = 2;
+    
+    JLabel worldTypeJLabel = new JLabel(String.format("<html><b>ServerType</b></html>", i));
```

Comment:
I like the placement of this & it functions fine, but you or me will have to reimplement the execution of this. A dropdown menu with english names would be better than text field for int input. Would not need worldTypeJLabel anymore after that

Reply:
actually tbh I don't think it functions for input, only for display. I change the int & click apply, it goes back to 1. I think because it's re-reading the world file.

Reply:
I fixed this.